stellar-multitenant/internal/provisioning/manager.go

```diff
@@ -134,32 +148,32 @@ func (m *Manager) handleProvisioningError(ctx context.Context, err error, t *ten
 	return provisioningErr
 }
 
-func (m *Manager) provisionTenant(ctx context.Context, pt *ProvisionTenant) (*tenant.Tenant, *message.Message, error) {
```

Comment:
Can we use named return variables here? By having a string being returned from provisionTenant is hard to grasp what that string means.